lib/Activity/Provider.php

```diff
@@ -78,7 +78,7 @@ public function parse($language, IEvent $event, IEvent $previousEvent = null) {
 				$event->setSubject($l->t('You removed an WebAuthn hardware token'));
 				break;
 			case 'webauthn_disabled_by_admin':
-				$event->setSubject($l->t('WebAuthn disabled by an admin'));
+				$event->setSubject($l->t('WebAuthn disabled by an administrator'));
```

Comment:
'WebAuthn disabled by an administrator' -> 'WebAuthn disabled by the administration'